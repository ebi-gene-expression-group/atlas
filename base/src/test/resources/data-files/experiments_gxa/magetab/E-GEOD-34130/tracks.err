[WARNING] Found 243 duplicates. E.g., AT1G01320
[WARNING] Found 243 duplicates. E.g., AT1G01320
[WARNING] Found 243 duplicates. E.g., AT1G01320
[WARNING] Found 243 duplicates. E.g., AT1G01320
[WARNING] Found 243 duplicates. E.g., AT1G01320
[WARNING] Found 243 duplicates. E.g., AT1G01320
[WARNING] Found 243 duplicates. E.g., AT1G01320
[WARNING] Found 243 duplicates. E.g., AT1G01320
[WARNING] Found 243 duplicates. E.g., AT1G01320
[WARNING] Found 243 duplicates. E.g., AT1G01320
[WARNING] Found 243 duplicates. E.g., AT1G01320
[WARNING] Found 243 duplicates. E.g., AT1G01320
[WARNING] Found 243 duplicates. E.g., AT1G01320
[WARNING] Found 243 duplicates. E.g., AT1G01320
