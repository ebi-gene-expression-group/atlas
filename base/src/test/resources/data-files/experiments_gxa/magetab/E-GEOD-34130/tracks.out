[INFO] GFF file loaded: 33602 entries.
[INFO] Loading /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Loaded /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Columns:Gene ID
 [INFO] Columns:Gene Name
 [INFO] Columns:Design Element
 [INFO] Columns:g10_g4.p-value
 [INFO] Columns:g10_g4.t-statistic
 [INFO] Columns:g10_g4.log2foldchange
 [INFO] Columns:g11_g5.p-value
 [INFO] Columns:g11_g5.t-statistic
 [INFO] Columns:g11_g5.log2foldchange
 [INFO] Columns:g12_g6.p-value
 [INFO] Columns:g12_g6.t-statistic
 [INFO] Columns:g12_g6.log2foldchange
 [INFO] Columns:g13_g7.p-value
 [INFO] Columns:g13_g7.t-statistic
 [INFO] Columns:g13_g7.log2foldchange
 [INFO] Columns:g14_g8.p-value
 [INFO] Columns:g14_g8.t-statistic
 [INFO] Columns:g14_g8.log2foldchange
 [INFO] Columns:g2_g1.p-value
 [INFO] Columns:g2_g1.t-statistic
 [INFO] Columns:g2_g1.log2foldchange
 [INFO] Columns:g9_g3.p-value
 [INFO] Columns:g9_g3.t-statistic
 [INFO] Columns:g9_g3.log2foldchange
[INFO] numeric
[INFO] numeric
[INFO] Excluding duplicates
[INFO] #Features with data:21122
[INFO] # values:21122
[INFO] Found 0 NA values and 0 +-Inf
[INFO] All done!
[INFO] GFF file loaded: 33602 entries.
[INFO] Loading /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Loaded /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Columns:Gene ID
 [INFO] Columns:Gene Name
 [INFO] Columns:Design Element
 [INFO] Columns:g10_g4.p-value
 [INFO] Columns:g10_g4.t-statistic
 [INFO] Columns:g10_g4.log2foldchange
 [INFO] Columns:g11_g5.p-value
 [INFO] Columns:g11_g5.t-statistic
 [INFO] Columns:g11_g5.log2foldchange
 [INFO] Columns:g12_g6.p-value
 [INFO] Columns:g12_g6.t-statistic
 [INFO] Columns:g12_g6.log2foldchange
 [INFO] Columns:g13_g7.p-value
 [INFO] Columns:g13_g7.t-statistic
 [INFO] Columns:g13_g7.log2foldchange
 [INFO] Columns:g14_g8.p-value
 [INFO] Columns:g14_g8.t-statistic
 [INFO] Columns:g14_g8.log2foldchange
 [INFO] Columns:g2_g1.p-value
 [INFO] Columns:g2_g1.t-statistic
 [INFO] Columns:g2_g1.log2foldchange
 [INFO] Columns:g9_g3.p-value
 [INFO] Columns:g9_g3.t-statistic
 [INFO] Columns:g9_g3.log2foldchange
[INFO] numeric
[INFO] Excluding duplicates
[INFO] #Features with data:21122
[INFO] # values:21122
[INFO] Found 0 NA values and 0 +-Inf
[INFO] All done!
[INFO] GFF file loaded: 33602 entries.
[INFO] Loading /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Loaded /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Columns:Gene ID
 [INFO] Columns:Gene Name
 [INFO] Columns:Design Element
 [INFO] Columns:g10_g4.p-value
 [INFO] Columns:g10_g4.t-statistic
 [INFO] Columns:g10_g4.log2foldchange
 [INFO] Columns:g11_g5.p-value
 [INFO] Columns:g11_g5.t-statistic
 [INFO] Columns:g11_g5.log2foldchange
 [INFO] Columns:g12_g6.p-value
 [INFO] Columns:g12_g6.t-statistic
 [INFO] Columns:g12_g6.log2foldchange
 [INFO] Columns:g13_g7.p-value
 [INFO] Columns:g13_g7.t-statistic
 [INFO] Columns:g13_g7.log2foldchange
 [INFO] Columns:g14_g8.p-value
 [INFO] Columns:g14_g8.t-statistic
 [INFO] Columns:g14_g8.log2foldchange
 [INFO] Columns:g2_g1.p-value
 [INFO] Columns:g2_g1.t-statistic
 [INFO] Columns:g2_g1.log2foldchange
 [INFO] Columns:g9_g3.p-value
 [INFO] Columns:g9_g3.t-statistic
 [INFO] Columns:g9_g3.log2foldchange
[INFO] numeric
[INFO] numeric
[INFO] Excluding duplicates
[INFO] #Features with data:21122
[INFO] # values:21122
[INFO] Found 19887 NA values and 0 +-Inf
[INFO] Entries with NAs removed
[INFO] All done!
[INFO] GFF file loaded: 33602 entries.
[INFO] Loading /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Loaded /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Columns:Gene ID
 [INFO] Columns:Gene Name
 [INFO] Columns:Design Element
 [INFO] Columns:g10_g4.p-value
 [INFO] Columns:g10_g4.t-statistic
 [INFO] Columns:g10_g4.log2foldchange
 [INFO] Columns:g11_g5.p-value
 [INFO] Columns:g11_g5.t-statistic
 [INFO] Columns:g11_g5.log2foldchange
 [INFO] Columns:g12_g6.p-value
 [INFO] Columns:g12_g6.t-statistic
 [INFO] Columns:g12_g6.log2foldchange
 [INFO] Columns:g13_g7.p-value
 [INFO] Columns:g13_g7.t-statistic
 [INFO] Columns:g13_g7.log2foldchange
 [INFO] Columns:g14_g8.p-value
 [INFO] Columns:g14_g8.t-statistic
 [INFO] Columns:g14_g8.log2foldchange
 [INFO] Columns:g2_g1.p-value
 [INFO] Columns:g2_g1.t-statistic
 [INFO] Columns:g2_g1.log2foldchange
 [INFO] Columns:g9_g3.p-value
 [INFO] Columns:g9_g3.t-statistic
 [INFO] Columns:g9_g3.log2foldchange
[INFO] numeric
[INFO] Excluding duplicates
[INFO] #Features with data:21122
[INFO] # values:21122
[INFO] Found 0 NA values and 0 +-Inf
[INFO] All done!
[INFO] GFF file loaded: 33602 entries.
[INFO] Loading /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Loaded /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Columns:Gene ID
 [INFO] Columns:Gene Name
 [INFO] Columns:Design Element
 [INFO] Columns:g10_g4.p-value
 [INFO] Columns:g10_g4.t-statistic
 [INFO] Columns:g10_g4.log2foldchange
 [INFO] Columns:g11_g5.p-value
 [INFO] Columns:g11_g5.t-statistic
 [INFO] Columns:g11_g5.log2foldchange
 [INFO] Columns:g12_g6.p-value
 [INFO] Columns:g12_g6.t-statistic
 [INFO] Columns:g12_g6.log2foldchange
 [INFO] Columns:g13_g7.p-value
 [INFO] Columns:g13_g7.t-statistic
 [INFO] Columns:g13_g7.log2foldchange
 [INFO] Columns:g14_g8.p-value
 [INFO] Columns:g14_g8.t-statistic
 [INFO] Columns:g14_g8.log2foldchange
 [INFO] Columns:g2_g1.p-value
 [INFO] Columns:g2_g1.t-statistic
 [INFO] Columns:g2_g1.log2foldchange
 [INFO] Columns:g9_g3.p-value
 [INFO] Columns:g9_g3.t-statistic
 [INFO] Columns:g9_g3.log2foldchange
[INFO] numeric
[INFO] numeric
[INFO] Excluding duplicates
[INFO] #Features with data:21122
[INFO] # values:21122
[INFO] Found 0 NA values and 0 +-Inf
[INFO] All done!
[INFO] GFF file loaded: 33602 entries.
[INFO] Loading /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Loaded /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Columns:Gene ID
 [INFO] Columns:Gene Name
 [INFO] Columns:Design Element
 [INFO] Columns:g10_g4.p-value
 [INFO] Columns:g10_g4.t-statistic
 [INFO] Columns:g10_g4.log2foldchange
 [INFO] Columns:g11_g5.p-value
 [INFO] Columns:g11_g5.t-statistic
 [INFO] Columns:g11_g5.log2foldchange
 [INFO] Columns:g12_g6.p-value
 [INFO] Columns:g12_g6.t-statistic
 [INFO] Columns:g12_g6.log2foldchange
 [INFO] Columns:g13_g7.p-value
 [INFO] Columns:g13_g7.t-statistic
 [INFO] Columns:g13_g7.log2foldchange
 [INFO] Columns:g14_g8.p-value
 [INFO] Columns:g14_g8.t-statistic
 [INFO] Columns:g14_g8.log2foldchange
 [INFO] Columns:g2_g1.p-value
 [INFO] Columns:g2_g1.t-statistic
 [INFO] Columns:g2_g1.log2foldchange
 [INFO] Columns:g9_g3.p-value
 [INFO] Columns:g9_g3.t-statistic
 [INFO] Columns:g9_g3.log2foldchange
[INFO] numeric
[INFO] Excluding duplicates
[INFO] #Features with data:21122
[INFO] # values:21122
[INFO] Found 0 NA values and 0 +-Inf
[INFO] All done!
[INFO] GFF file loaded: 33602 entries.
[INFO] Loading /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Loaded /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Columns:Gene ID
 [INFO] Columns:Gene Name
 [INFO] Columns:Design Element
 [INFO] Columns:g10_g4.p-value
 [INFO] Columns:g10_g4.t-statistic
 [INFO] Columns:g10_g4.log2foldchange
 [INFO] Columns:g11_g5.p-value
 [INFO] Columns:g11_g5.t-statistic
 [INFO] Columns:g11_g5.log2foldchange
 [INFO] Columns:g12_g6.p-value
 [INFO] Columns:g12_g6.t-statistic
 [INFO] Columns:g12_g6.log2foldchange
 [INFO] Columns:g13_g7.p-value
 [INFO] Columns:g13_g7.t-statistic
 [INFO] Columns:g13_g7.log2foldchange
 [INFO] Columns:g14_g8.p-value
 [INFO] Columns:g14_g8.t-statistic
 [INFO] Columns:g14_g8.log2foldchange
 [INFO] Columns:g2_g1.p-value
 [INFO] Columns:g2_g1.t-statistic
 [INFO] Columns:g2_g1.log2foldchange
 [INFO] Columns:g9_g3.p-value
 [INFO] Columns:g9_g3.t-statistic
 [INFO] Columns:g9_g3.log2foldchange
[INFO] numeric
[INFO] numeric
[INFO] Excluding duplicates
[INFO] #Features with data:21122
[INFO] # values:21122
[INFO] Found 0 NA values and 0 +-Inf
[INFO] All done!
[INFO] GFF file loaded: 33602 entries.
[INFO] Loading /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Loaded /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Columns:Gene ID
 [INFO] Columns:Gene Name
 [INFO] Columns:Design Element
 [INFO] Columns:g10_g4.p-value
 [INFO] Columns:g10_g4.t-statistic
 [INFO] Columns:g10_g4.log2foldchange
 [INFO] Columns:g11_g5.p-value
 [INFO] Columns:g11_g5.t-statistic
 [INFO] Columns:g11_g5.log2foldchange
 [INFO] Columns:g12_g6.p-value
 [INFO] Columns:g12_g6.t-statistic
 [INFO] Columns:g12_g6.log2foldchange
 [INFO] Columns:g13_g7.p-value
 [INFO] Columns:g13_g7.t-statistic
 [INFO] Columns:g13_g7.log2foldchange
 [INFO] Columns:g14_g8.p-value
 [INFO] Columns:g14_g8.t-statistic
 [INFO] Columns:g14_g8.log2foldchange
 [INFO] Columns:g2_g1.p-value
 [INFO] Columns:g2_g1.t-statistic
 [INFO] Columns:g2_g1.log2foldchange
 [INFO] Columns:g9_g3.p-value
 [INFO] Columns:g9_g3.t-statistic
 [INFO] Columns:g9_g3.log2foldchange
[INFO] numeric
[INFO] Excluding duplicates
[INFO] #Features with data:21122
[INFO] # values:21122
[INFO] Found 0 NA values and 0 +-Inf
[INFO] All done!
[INFO] GFF file loaded: 33602 entries.
[INFO] Loading /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Loaded /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Columns:Gene ID
 [INFO] Columns:Gene Name
 [INFO] Columns:Design Element
 [INFO] Columns:g10_g4.p-value
 [INFO] Columns:g10_g4.t-statistic
 [INFO] Columns:g10_g4.log2foldchange
 [INFO] Columns:g11_g5.p-value
 [INFO] Columns:g11_g5.t-statistic
 [INFO] Columns:g11_g5.log2foldchange
 [INFO] Columns:g12_g6.p-value
 [INFO] Columns:g12_g6.t-statistic
 [INFO] Columns:g12_g6.log2foldchange
 [INFO] Columns:g13_g7.p-value
 [INFO] Columns:g13_g7.t-statistic
 [INFO] Columns:g13_g7.log2foldchange
 [INFO] Columns:g14_g8.p-value
 [INFO] Columns:g14_g8.t-statistic
 [INFO] Columns:g14_g8.log2foldchange
 [INFO] Columns:g2_g1.p-value
 [INFO] Columns:g2_g1.t-statistic
 [INFO] Columns:g2_g1.log2foldchange
 [INFO] Columns:g9_g3.p-value
 [INFO] Columns:g9_g3.t-statistic
 [INFO] Columns:g9_g3.log2foldchange
[INFO] numeric
[INFO] numeric
[INFO] Excluding duplicates
[INFO] #Features with data:21122
[INFO] # values:21122
[INFO] Found 21122 NA values and 0 +-Inf
[INFO] Entries with NAs removed
[INFO] All done!
[INFO] GFF file loaded: 33602 entries.
[INFO] Loading /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Loaded /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Columns:Gene ID
 [INFO] Columns:Gene Name
 [INFO] Columns:Design Element
 [INFO] Columns:g10_g4.p-value
 [INFO] Columns:g10_g4.t-statistic
 [INFO] Columns:g10_g4.log2foldchange
 [INFO] Columns:g11_g5.p-value
 [INFO] Columns:g11_g5.t-statistic
 [INFO] Columns:g11_g5.log2foldchange
 [INFO] Columns:g12_g6.p-value
 [INFO] Columns:g12_g6.t-statistic
 [INFO] Columns:g12_g6.log2foldchange
 [INFO] Columns:g13_g7.p-value
 [INFO] Columns:g13_g7.t-statistic
 [INFO] Columns:g13_g7.log2foldchange
 [INFO] Columns:g14_g8.p-value
 [INFO] Columns:g14_g8.t-statistic
 [INFO] Columns:g14_g8.log2foldchange
 [INFO] Columns:g2_g1.p-value
 [INFO] Columns:g2_g1.t-statistic
 [INFO] Columns:g2_g1.log2foldchange
 [INFO] Columns:g9_g3.p-value
 [INFO] Columns:g9_g3.t-statistic
 [INFO] Columns:g9_g3.log2foldchange
[INFO] numeric
[INFO] Excluding duplicates
[INFO] #Features with data:21122
[INFO] # values:21122
[INFO] Found 0 NA values and 0 +-Inf
[INFO] All done!
[INFO] GFF file loaded: 33602 entries.
[INFO] Loading /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Loaded /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Columns:Gene ID
 [INFO] Columns:Gene Name
 [INFO] Columns:Design Element
 [INFO] Columns:g10_g4.p-value
 [INFO] Columns:g10_g4.t-statistic
 [INFO] Columns:g10_g4.log2foldchange
 [INFO] Columns:g11_g5.p-value
 [INFO] Columns:g11_g5.t-statistic
 [INFO] Columns:g11_g5.log2foldchange
 [INFO] Columns:g12_g6.p-value
 [INFO] Columns:g12_g6.t-statistic
 [INFO] Columns:g12_g6.log2foldchange
 [INFO] Columns:g13_g7.p-value
 [INFO] Columns:g13_g7.t-statistic
 [INFO] Columns:g13_g7.log2foldchange
 [INFO] Columns:g14_g8.p-value
 [INFO] Columns:g14_g8.t-statistic
 [INFO] Columns:g14_g8.log2foldchange
 [INFO] Columns:g2_g1.p-value
 [INFO] Columns:g2_g1.t-statistic
 [INFO] Columns:g2_g1.log2foldchange
 [INFO] Columns:g9_g3.p-value
 [INFO] Columns:g9_g3.t-statistic
 [INFO] Columns:g9_g3.log2foldchange
[INFO] numeric
[INFO] numeric
[INFO] Excluding duplicates
[INFO] #Features with data:21122
[INFO] # values:21122
[INFO] Found 0 NA values and 0 +-Inf
[INFO] All done!
[INFO] GFF file loaded: 33602 entries.
[INFO] Loading /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Loaded /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Columns:Gene ID
 [INFO] Columns:Gene Name
 [INFO] Columns:Design Element
 [INFO] Columns:g10_g4.p-value
 [INFO] Columns:g10_g4.t-statistic
 [INFO] Columns:g10_g4.log2foldchange
 [INFO] Columns:g11_g5.p-value
 [INFO] Columns:g11_g5.t-statistic
 [INFO] Columns:g11_g5.log2foldchange
 [INFO] Columns:g12_g6.p-value
 [INFO] Columns:g12_g6.t-statistic
 [INFO] Columns:g12_g6.log2foldchange
 [INFO] Columns:g13_g7.p-value
 [INFO] Columns:g13_g7.t-statistic
 [INFO] Columns:g13_g7.log2foldchange
 [INFO] Columns:g14_g8.p-value
 [INFO] Columns:g14_g8.t-statistic
 [INFO] Columns:g14_g8.log2foldchange
 [INFO] Columns:g2_g1.p-value
 [INFO] Columns:g2_g1.t-statistic
 [INFO] Columns:g2_g1.log2foldchange
 [INFO] Columns:g9_g3.p-value
 [INFO] Columns:g9_g3.t-statistic
 [INFO] Columns:g9_g3.log2foldchange
[INFO] numeric
[INFO] Excluding duplicates
[INFO] #Features with data:21122
[INFO] # values:21122
[INFO] Found 0 NA values and 0 +-Inf
[INFO] All done!
[INFO] GFF file loaded: 33602 entries.
[INFO] Loading /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Loaded /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Columns:Gene ID
 [INFO] Columns:Gene Name
 [INFO] Columns:Design Element
 [INFO] Columns:g10_g4.p-value
 [INFO] Columns:g10_g4.t-statistic
 [INFO] Columns:g10_g4.log2foldchange
 [INFO] Columns:g11_g5.p-value
 [INFO] Columns:g11_g5.t-statistic
 [INFO] Columns:g11_g5.log2foldchange
 [INFO] Columns:g12_g6.p-value
 [INFO] Columns:g12_g6.t-statistic
 [INFO] Columns:g12_g6.log2foldchange
 [INFO] Columns:g13_g7.p-value
 [INFO] Columns:g13_g7.t-statistic
 [INFO] Columns:g13_g7.log2foldchange
 [INFO] Columns:g14_g8.p-value
 [INFO] Columns:g14_g8.t-statistic
 [INFO] Columns:g14_g8.log2foldchange
 [INFO] Columns:g2_g1.p-value
 [INFO] Columns:g2_g1.t-statistic
 [INFO] Columns:g2_g1.log2foldchange
 [INFO] Columns:g9_g3.p-value
 [INFO] Columns:g9_g3.t-statistic
 [INFO] Columns:g9_g3.log2foldchange
[INFO] numeric
[INFO] numeric
[INFO] Excluding duplicates
[INFO] #Features with data:21122
[INFO] # values:21122
[INFO] Found 0 NA values and 0 +-Inf
[INFO] All done!
[INFO] GFF file loaded: 33602 entries.
[INFO] Loading /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Loaded /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130/E-GEOD-34130_A-AFFY-2-analytics.tsv
[INFO] Columns:Gene ID
 [INFO] Columns:Gene Name
 [INFO] Columns:Design Element
 [INFO] Columns:g10_g4.p-value
 [INFO] Columns:g10_g4.t-statistic
 [INFO] Columns:g10_g4.log2foldchange
 [INFO] Columns:g11_g5.p-value
 [INFO] Columns:g11_g5.t-statistic
 [INFO] Columns:g11_g5.log2foldchange
 [INFO] Columns:g12_g6.p-value
 [INFO] Columns:g12_g6.t-statistic
 [INFO] Columns:g12_g6.log2foldchange
 [INFO] Columns:g13_g7.p-value
 [INFO] Columns:g13_g7.t-statistic
 [INFO] Columns:g13_g7.log2foldchange
 [INFO] Columns:g14_g8.p-value
 [INFO] Columns:g14_g8.t-statistic
 [INFO] Columns:g14_g8.log2foldchange
 [INFO] Columns:g2_g1.p-value
 [INFO] Columns:g2_g1.t-statistic
 [INFO] Columns:g2_g1.log2foldchange
 [INFO] Columns:g9_g3.p-value
 [INFO] Columns:g9_g3.t-statistic
 [INFO] Columns:g9_g3.log2foldchange
[INFO] numeric
[INFO] Excluding duplicates
[INFO] #Features with data:21122
[INFO] # values:21122
[INFO] Found 0 NA values and 0 +-Inf
[INFO] All done!

------------------------------------------------------------
Sender: LSF System <lsf@ebi5-042>
Subject: Job 1032376: </nfs/production3/ma/home/atlas3-production/sw/atlasinstall_prod/atlasprod/irap/gxa_generate_tracks_for_experiment.sh  /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130  > in cluster <EBI> Done

Job </nfs/production3/ma/home/atlas3-production/sw/atlasinstall_prod/atlasprod/irap/gxa_generate_tracks_for_experiment.sh  /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130  > was submitted from host <ebi1-121> by user <fg_atlas> in cluster <EBI>.
Job was executed on host(s) <ebi5-042>, in queue <production-rh6>, as user <fg_atlas> in cluster <EBI>.
</net/isilonP/public/rw/homes/fg_atlas> was used as the home directory.
</nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130> was used as the working directory.
Started at Tue Apr 12 17:28:24 2016
Results reported at Tue Apr 12 17:32:41 2016

Your job looked like:

------------------------------------------------------------
# LSBATCH: User input
/nfs/production3/ma/home/atlas3-production/sw/atlasinstall_prod/atlasprod/irap/gxa_generate_tracks_for_experiment.sh  /nfs/production3/ma/home/atlas3-production/analysis/differential/microarray/experiments/E-GEOD-34130  
------------------------------------------------------------

Successfully completed.

Resource usage summary:

    CPU time :               250.59 sec.
    Max Memory :             545 MB
    Average Memory :         278.23 MB
    Total Requested Memory : 4096.00 MB
    Delta Memory :           3551.00 MB
    (Delta: the difference between total requested memory and actual max usage.)
    Max Swap :               1040 MB

    Max Processes :          5
    Max Threads :            6

The output (if any) is above this job summary.



PS:

Read file <tracks.err> for stderr output of this job.

